src/server/Html.jsx

```diff
@@ -25,13 +25,18 @@ export default class Html extends Component {
       // in dev we separate vendor out for faster webpack rebuild times.
       return [
         <script key="vendor" src={assets.javascript.vendor} charSet="UTF-8" />,
+        // TODO: we can switch to mapbox if we want.
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-beta.2/leaflet.js" charSet="UTF-8" />,
+        <script src="https://mapzen.com/tangram/0.6/tangram.min.js" charSet="UTF-8" />,
```

Comment:
Let's stick with what you know/are interested in using and if we find at the end there is a simple way to swap for less kb, we can consider it.